packages/nvidia_nat_core/tests/nat/utils/test_atif_converter.py

```
@@ -0,0 +1,507 @@
+    ):
+        """Stream converter produces the same steps as batch converter."""
+        batch_result = batch_converter.convert(simple_trajectory, session_id="test")
+
+        stream_conv = ATIFStreamConverter()
+        for ist in simple_trajectory:
+            stream_conv.push(ist)
+        stream_conv.finalize()
+        stream_result = stream_conv.get_trajectory()
+
+        assert len(stream_result.steps) == len(batch_result.steps)
+        for s_step, b_step in zip(stream_result.steps, batch_result.steps):
```

Ruff (0.15.2)

[warning] `zip()` without an explicit `strict=` parameter

Add explicit value for parameter `strict=`

(B905)